### PR TITLE
Update tutorials - Candlestick chart

### PR DESCRIPTION
**Pull request by vulh-infragistics**

### CODE REVIEW

File: docs/user/images/add-visualization-button.png
Comment (brianlagunas) [inline docs/user/images/add-visualization-button.png:1]: this image is difficult to see/read

File: docs/user/images/candlestick-chart-types.png
Comment (brianlagunas) [inline docs/user/images/candlestick-chart-types.png:1]: The candle stick chart type is not visible in the Chart Types dropdown. Maybe bring attention to it by placing a red rectangle around it in the drop down.

**Review by brianlagunas — CHANGES_REQUESTED** [review]

File: docs/user/images/add-visualization-button.png
Comment (vulh-infragistics) [reply docs/user/images/add-visualization-button.png:1]: I have replaced with bigger image

File: docs/user/images/candlestick-chart-types.png
Comment (vulh-infragistics) [reply docs/user/images/candlestick-chart-types.png:1]: Added

**Review by brianlagunas — APPROVED** [review]

**Pull request merged**